lib/_debugger.js

```diff
@@ -735,7 +735,7 @@ function Interface(stdin, stdout, args) {
     prompt: 'debug> ',
     input: this.stdin,
     output: this.stdout,
-    eval: this.controlEval.bind(this),
+    eval: (code, context, filename, callback) => this.controlEval(),
```

Comment:
Also myou can probably shorten these really long ones to `(code, ctx, file, cb)`